tests/integration/test_pip.py

```diff
@@ -89,3 +91,92 @@ def test_pip_packages(
     _ = utils.fetch_deps_and_check_output(
         tmpdir, test_case, test_params, source_folder, test_data_dir, cachi2_image
     )
+
+
+@pytest.mark.parametrize(
+    "test_params,check_cmd",
+    [
+        # Test case checks fetching pip dependencies, generating environment vars file,
+        # building image with all prepared prerequisites and testing if pip packages are present
+        # in built image
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/pip-e2e-test.git",
```

Comment:
TLDR, I don't think this makes a good base for the e2e test :cry: 

It's a weird something that tests OSBS, but it's not a properly constructed Python project

Reply:
I would probably suggest starting a new one from scratch


1. Make it depend on `requests` + something simple installed from a url (like dockerfile-parse, pyarn, gemlock-parser)
2. Make it a real, installable python package that imports the things it depends on
3. Set up requirements.txt and requirements-build.txt according to the docs
   * requirements-build.txt should include `flit-core`
4. Bonus: use hashes for at least one of the requirements files
5. Install in the container build, verify that the installed app works

Reply:
Looks much better now :tada: